docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -522,27 +537,28 @@ This is done by ensuring tokens can be trusted.
+The verification process of OIDC authorization code flow tokens follows the Bearer token authentication token verification and introspection logic.
+For more information, see the xref:security-oidc-bearer-token-authentication.adoc#token-verification-introspection[Token verification and introspection] section of the "Quarkus OpenID Connect (OIDC) Bearer token authentication" guide.
 
 [NOTE]
---
-With Quarkus `web-app` applications, only the `IdToken` is verified by default because the access token is not used to access the current Quarkus `web-app` endpoint and is intended to be propagated to the services expecting this access token.
-If you expect the access token to contain the roles required to access the current Quarkus endpoint (`quarkus.oidc.roles.source=accesstoken`) then it will also be verified.
---
+====
+With Quarkus `web-app` applications, only the `IdToken` is verified by default because the access token is not used to access the current Quarkus web-app endpoint and is intended to be propagated to the services expecting this access token.
+If you expect the access token to contain the roles required to access the current Quarkus endpoint (`quarkus.oidc.roles.source=accesstoken`), then it will also be verified.
+====
 
 [[token-introspection-userinfo-cache]]
 ==== Token introspection and UserInfo cache
 
 Code flow access tokens are not introspected unless they are expected to be the source of roles.
-They will however be used to get `UserInfo`.
+However, they will be used to get `UserInfo`.
 There will be one or two remote calls with the code flow access token, if the token introspection, `UserInfo`, or both are required.
 
-For more information about using a default token cache or registering a custom cache implementation, see xref:security-oidc-bearer-token-authentication.adoc#token-introspection-userinfo-cache[Token Introspection and UserInfo cache].
+For more information about using a default token cache or registering a custom cache implementation, see xref:security-oidc-bearer-token-authentication.adoc#token-introspection-userinfo-cache[Token introspection and UserInfo cache].
 
 [[jwt-claim-verification]]
 ==== JSON web token claim verification
 
-For information about the claim verification, including the `iss` (issuer) claim, see the xref:security-oidc-bearer-token-authentication.adoc#jwt-claim-verification[JSON Web Token Claim verification] section.
+For information about the claim verification, including the `iss` (issuer) claim, see the xref:security-oidc-bearer-token-authentication.adoc#jwt-claim-verification[JSON web token claim verification] section.
 It applies to ID tokens but also to access tokens in a JWT format if the `web-app` application has requested the access token verification.
```

Comment:
```suggestion
It applies to ID and access tokens in a JWT format if the `web-app` application has requested the access token verification.
```

Reply:
@MichalMaler - Modifying this further slightly as it could be confusing an imply that both the "ID tokens" and the "access tokens" are in a JWT format. My understanding of the original sentence is that only the "access tokens" are in a JWT format.
Proposing instead to change to the following: "It applies to ID tokens and **also to* access tokens in a JWT format, if the `web-app` application has requested the access token verification."